README.rst

```diff
@@ -52,6 +52,11 @@ Installing the released version can be done using pip::
 
    pip install jdaviz --upgrade
 
+You may want to consider installing ``jdaviz`` in a new virtual or conda environment to avoid version conflicts with other packages you may have installed, for example::
```

Comment:
Should we move this blurb to before the `pip install` part since this needs to happen first?

Also, break the long line?

```suggestion
You may want to consider installing ``jdaviz`` in a new virtual or conda environment
to avoid version conflicts with other packages you may have installed, for example::
```

Reply:
hmmm that's a good point.  [RTD](https://jdaviz.readthedocs.io/en/latest/installation.html) has separate sections for environment and pip install... but I don't think we want to re-include everything here.  I saw this more as a "footnote", but can see how it could cause confusion for anyone who ran the commands in the order they are here, so would be okay with re-ordering them as long as no one thinks it "buries" the pip line too deep, since that is most important? 🤷

Reply:
(I changed the order and added the line-break - if that works for you, it works for me!)

Reply:
I am guilty of running instructions in order... 😹

Reply:
I think this makes more sense as long as we don't add much more before the pip line, so let's go with this for now.  Thanks for the suggestion!